<launch>

<node pkg="yolo_tf" type="visualize.py" name="yolo_vis" args="$(find yolo_tf)/data/coco.names">
      <remap from="image" to="/hsrb/head_rgbd_sensor/rgb/image_rect_color"/>
</node>
</launch>
